book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,97 @@
+
+**Principles of emergent strategy**
+*	Small is good, small is all. (The large is a reflection of the small)
+*	Change is constant (be like water)
+*	There is always enough time for the right work. 
+*	There is a conversation in the room that only this moment can have. Find it
+*	Never a failure, always a lesson
+*	Trust the people (If you trust the people they become trustworthy or the boundaries will become clear)
+*	Move at the speed of trust. Focus on critical connections more than critical mass – build the resilience by building the relationships 
+*	Less prep, more presence
+*	What you pay attention to grows
+
+Conversation is a crucial way to explore what we believe and to make new understandings and ideas possible. 
+Consider the areas where trust needs to be built: you will need to demonstrate that you're thinking about the same values/concerns. 
+To accomplish your vision for the future it needs to be clear what needs to be done and who will do it. 
+There should be time for reflection and evaluation to apply lessons learned to future work, and time for celebration of collective achievements. 
+
+(er-activism-cultural-change-transitions)=
+## Managing Transitions
+
+Managing Transitions by William Bridges teaches us that a change will always be accompanied by a transition. A transition begins when something needs to be let go, and sits in between the old and the new isutation. This transition phase can be insecure for people who will have to adjust to a new situation, but also an opportunity for innovation to happen if people are adequately resourced and supported (by policies, groupings, procedures). 
+
+To manage transitions it needs to be clear what the problem (reason for change) is. People will not change if they are not experiencing the problem and are convinced something needs to be adjusted. Those experiencing the problem will also have expertise and knowledge to come up with an effective solution. Individuals will stand to loose something when change is happening and a sense of control over this transition may help making transition phases less uncomfortable. It is important to learn what problems people are facing in a trancition face, and recognise individuals that do contribute to the change. People will need to feel safe to experiment and take risks, and not be punished when these experiments fail.
+
+It is important that changes are meaningfully clustered: avoid further changes to avoid a messy change pile. 
+
+Short term goals can give people a sense of achievement and encourage them to move forward.  
+
+To ensure that people will change their behaviour, they will need to learn how to adapt to new behaviours through training. People will not be able to invest in this, if the new behaviours are not adequately rewarded (for example, focused placed on publications written instead of open data/software).
+
+There are four P's in transitions (see Managing Transitions by William Bridges): 
```

Comment:
Think this $ Ps summary would be better up the top of this section

Reply:
Even before the introduction into transitions?